### DISCUSSION TITLE
Get current time

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17096] [How to] [answered]
Workflows expression get_date('today') returning date but not date and current time such as: 20-05-2023 00:00 
How to get current time according to user settings?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
here you can easily add it,
https://github.com/YetiForceCompany/YetiForceCRM/blob/37a4b61f37bd933ef7f9c2a6d916cf046c92232f/modules/com_vtiger_workflow/expression_engine/VTExpressionEvaluater.php#L174-L187

and then here https://github.com/YetiForceCompany/YetiForceCRM/pulls :)
